arch/xtensa/src/esp32/esp32_wtd.h

```diff
@@ -41,7 +41,6 @@
 #define ESP32_WTD_STOP(d)                       ((d)->ops->stop(d))
 #define ESP32_WTD_LOCK(d)                       ((d)->ops->enablewp(d))
 #define ESP32_WTD_UNLOCK(d)                     ((d)->ops->disablewp(d))
-#define ESP32_WTD_INITCONF(d)                   ((d)->ops->initconf(d))
```

Comment:
should we remove initconf from esp32_wtd_ops_s too?

Reply:
Yes. Definitely! 
Thanks for pointing it out to me.
I will fix it now.

Reply:
Done